src/pages/Staking/index.tsx

```diff
@@ -575,7 +577,11 @@ const Staking = (): JSX.Element => {
       throw new Error('Something went wrong!');
     }
 
-    return stakedAmountAndEndBlock.endBlock - currentBlockNumber;
+    return (
+      hasStakedAmount ?
+        stakedAmountAndEndBlock.endBlock - currentBlockNumber : // If the user has staked
+        null // If the user has not staked
+    );
```

Comment:
The value is properly returned depending on whether the user has staked or not.
It had an bug because `stakedAmountAndEndBlock.endBlock` was zero when the user had not staked and that was not considered.
Now `remainingBlockNumbersToUnstake` is null when the user has not staked. It was originally negative number which was wrong.